docs/releases/4.0.2.txt

```diff
@@ -11,3 +11,6 @@ Bugfixes
 
 * Fixed a bug in Django 4.0 where ``TestCase.captureOnCommitCallbacks()`` could
   execute callbacks multiple times (:ticket:`33410`).
+
+* Fixed a bug in Django 4.0 where ``help_text`` was not escaped in form
```

Comment:
`was not escaped` -> `was escaped`

Reply:
Your wording here is much better. Thank you.